File: insertion-sort.py
def insertion_sort(a):
    i = 1
    while i < len(a):
        j = i
        while j > 0 and a[j-1] > a[j]:
            a[j], a[j-1] = a[j-1], a[j]
            j -= 1
        i += 1

File: insertion-sort.js
function insertionSort(a) {
    let i = 1;
    while (i < a.length) {
      let j = i;
      while (j > 0 && a[j-1] > a[j]) {
        const tmp = a[j];
        a[j] = a[j - 1];
        a[j - 1] = tmp;
        j -= 1;
      }
      i += 1;
    }
  }
